package.json

```diff
@@ -29,86 +28,24 @@
     "url": "https://github.com/simularium/nbsv"
   },
   "scripts": {
-    "build": "yarn run build:lib && yarn run build:nbextension && yarn run build:labextension:dev",
-    "build:prod": "yarn run build:lib && yarn run build:nbextension && yarn run build:labextension",
-    "build:labextension": "jupyter labextension build .",
-    "build:labextension:dev": "jupyter labextension build --development True .",
-    "build:lib": "tsc",
-    "build:nbextension": "webpack",
-    "clean": "yarn run clean:lib && yarn run clean:nbextension && yarn run clean:labextension",
-    "clean:lib": "rimraf lib",
-    "clean:labextension": "rimraf nbsv/labextension",
-    "clean:nbextension": "rimraf nbsv/nbextension/static/index.js",
-    "lint": "eslint . --ext .ts,.tsx --fix",
-    "lint:check": "eslint . --ext .ts,.tsx",
-    "prepack": "yarn run build:lib",
-    "test": "yarn jest --runInBand",
-    "watch": "npm-run-all -p watch:lib watch:nbextension watch:labextension",
-    "watch:lib": "tsc -w",
-    "watch:nbextension": "webpack --watch --mode=development",
-    "watch:labextension": "jupyter labextension watch ."
+    "watch": "yarn run build -- --sourcemap=inline --watch",
+    "build": "esbuild src/widget.tsx --bundle --minify --format=esm --outdir=./nbsv/static --asset-names=[name].[ext]",
+    "typecheck": "tsc --noEmit"
   },
   "dependencies": {
-    "@aics/simularium-viewer": "^3.7.0",
-    "@ant-design/icons": "4.0.0",
-    "@jupyter-widgets/base": "^1.1.10 || ^2 || ^3 || ^4 || ^5 || ^6",
-    "antd": "^5.4.2",
-    "crypto": "1.0.1",
-    "jest-cli": "^29.7.0",
-    "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "@aics/simularium-viewer": "file:../simularium-viewer",
```

Comment:
TODO: once the viewer has a new release, that should be used here (this line will currently cause the build to fail)